.gitlab-ci.yml

```diff
@@ -100,6 +105,41 @@ github:failure:
   except:
     - master
 
+mender-qa:start:
+  stage: .pre
+  # Keep overhead low by using a small image with curl preinstalled.
+  image: appropriate/curl
+  before_script:
+    - curl -Lf -o $CI_PROJECT_DIR/github_pull_request_status
+      https://github.com/mendersoftware/mender-qa/raw/$CI_COMMIT_SHA/scripts/github_pull_request_status
+    - chmod ugo+x $CI_PROJECT_DIR/github_pull_request_status
```

Comment:
We don't need to download it, right? This is the mender-qa pipeline so the script is already in `$CI_PROJECT_DIR/scripts/...`

Reply:
(same for `success` and `failure` jobs)

Reply:
Doh! Of course! I had it in my head that this would be "reused" for other repositories, but of course it all still happens in the mender-qa pipeline. Fixing!